x-pack/legacy/plugins/maps/public/shared/layers/sources/all_sources.js

```diff
@@ -27,3 +27,7 @@ export const ALL_SOURCES = [
   XYZTMSSource,
   WMSSource,
 ];
+
+export const BETA_SOURCES = [
```

Comment:
Instead of having to track beta sources with a constant, how about just having a static variable that specifies when a source is beta. So maybe add the property `GeojsonFileSource.isBeta` to GeojsonFileSource class